src/thermo/critProperties.xml

```diff
@@ -0,0 +1,6331 @@
+           <Vc value="307.5"> </Vc>
+           <Zc value="0.281"> </Zc>
+           <omega value="0.258"> </omega>
+       </species>
+       <species name= "C4H10S">
+           <ChemName name ="2-methyl-2-propanethiol"> </ChemName>
+           <MW value = "90.183"> </MW>
+           <Tc value= "521"> </Tc>
+           <Pc value= "37E6 "> </Pc>
+           <Vc value="302.5"> </Vc>
+           <Zc value="0.295"> </Zc>
+           <omega value="0.23"> </omega>
+       </species>
+       <species name= "C4H10S">
+           <ChemName name ="2  ethyl-disulfide"> </ChemName>
+           <MW value = "122.243"> </MW>
+           <Tc value= "642.1"> </Tc>
+           <Pc value= "35E6 "> </Pc>
+           <Vc value="367.5"> </Vc>
+           <Zc value="0.279"> </Zc>
+           <omega value="0.306"> </omega>
+       </species>
+       <species name= "C4H11N">
+           <ChemName name ="butylamine"> </ChemName>
+           <MW value = "73.138"> </MW>
+           <Tc value= "524"> </Tc>
+           <Pc value= "39.8E6 "> </Pc>
+           <Vc value="288.5"> </Vc>
+           <Zc value="0.277"> </Zc>
+           <omega value="0.329"> </omega>
+       </species>
+       <species name= "C4H11N">
+           <ChemName name ="sec-butylamine"> </ChemName>
+           <MW value = "73.138"> </MW>
+           <Tc value= "511"> </Tc>
+           <Pc value= "39.7E6 "> </Pc>
+           <Vc value="282.5"> </Vc>
+           <Zc value="0.283"> </Zc>
+           <omega value="0.368"> </omega>
+       </species>
+       <species name= "C4H11N">
+           <ChemName name ="tert-butylamine"> </ChemName>
+           <MW value = "73.138"> </MW>
+           <Tc value= "483.9"> </Tc>
+           <Pc value= "38.5E6 "> </Pc>
+           <Vc value="292.6"> </Vc>
+           <Zc value="0.279"> </Zc>
+           <omega value="0.285"> </omega>
+       </species>
+       <species name= "C4H11N">
+           <ChemName name ="diethylamine"> </ChemName>
+           <MW value = "73.138"> </MW>
+           <Tc value= "496.2"> </Tc>
+           <Pc value= "39.2E6 "> </Pc>
+           <Vc value="301"> </Vc>
+           <Zc value="0.27"> </Zc>
+           <omega value="0.291"> </omega>
+       </species>
+       <species name= "C5H5N">
+           <ChemName name ="pyridine"> </ChemName>
+           <MW value = "79.101"> </MW>
+           <Tc value= "618.7"> </Tc>
+           <Pc value= "58.8E6 "> </Pc>
+           <Vc value="252.8"> </Vc>
+           <Zc value="0.277"> </Zc>
+           <omega value="0.243"> </omega>
+       </species>
+       <species name= "C5H6S">
+           <ChemName name ="2-methylthiophene"> </ChemName>
+           <MW value = "98.162"> </MW>
+           <Tc value= "610"> </Tc>
+           <Pc value= "48.5E6 "> </Pc>
+           <Vc value="275.5"> </Vc>
+           <Zc value="0.265"> </Zc>
+           <omega value="0.238"> </omega>
+       </species>
+       <species name= "C5H6S">
+           <ChemName name ="3-methylthiophene"> <
```

Comment:
There are 7 entries for `C10H14`. Right now, only the first entry will ever be used, which is a little troubling. At a minimum, maybe we should specify an alternate name for the other isomers, e.g. `C10H14-1`, `C10H14-2`, etc. This applies to quite a few other species as well.